src/gen_module.sh

```diff
@@ -62,15 +77,15 @@ generated_mod_dir=${SCRIPT_DIR}/mods/generated_mods
 mkdir -p ${generated_mod_dir}
 ${blender_bin} --background --factory-startup -noaudio --python ${SCRIPT_DIR}/gen_modfile/gen_modules_modfile.py -- -m addon_utils -o ${generated_mod_dir}/gen_modules_modfile
 mkdir ${generated_mod_dir}/gen_startup_modfile
-python ${SCRIPT_DIR}/gen_modfile/gen_startup_modfile.py -i ${startup_dir} -o ${generated_mod_dir}/gen_startup_modfile/bpy.json
+${python} ${SCRIPT_DIR}/gen_modfile/gen_startup_modfile.py -i ${startup_dir} -o ${generated_mod_dir}/gen_startup_modfile/bpy.json
```

Comment:
Please rename this too as I mentioned before.

Reply:
see comment above, renamed